ui/pages/confirmations/send/send-content/__snapshots__/send-content.component.test.js.snap

```diff
@@ -287,3 +287,5 @@ exports[`SendContent Component render should match snapshot 1`] = `
   </div>
 </div>
 `;
+
+exports[`SendContent Component render should match snapshot 2`] = `<div />`;
```

Comment:
Not sure what this is, can most likely be reverted?

Reply:
this came due the merge from main and then updating the snapshots.